src/main/java/com/github/instagram4j/instagram4j/responses/tags/search/SearchTagResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.github.instagram4j.instagram4j.responses.tags.search;
+
+import com.github.instagram4j.instagram4j.responses.IGPaginatedResponse;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class SearchTagResponse extends IGPaginatedResponse {
```

Comment:
This response should be renamed to TagsSearchResponse and be under the "tags" package (Not the tags.search package)

Reply:
Fixed